cmd/security-secrets-setup/README.md

```diff
@@ -95,7 +99,7 @@ To see what the TLS materials are generated inside the docker container,
 one can run the built docker image above (assuming the version number is `1.1.0`) as:
 
 ```sh
-$ docker run --rm -it edgexfoundry/docker-edgex-vault:1.1.0-dev /bin/sh
+docker run --rm -it edgexfoundry/docker-edgex-vault:1.1.0-dev /bin/sh
```

Comment:
Whoops... This is out of date, or the container name is wrong.  We are using a stock "vault" image now instead of a custom one.  Probably needs to be updated to docker-edgex-secrets-setup-go:master-dev

Reply:
@bnevis-i Can I suggest a separate issue for this one? All I did was cleanup the command line so it was an easy copy/paste.

Reply:
Sure.